internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -33,13 +33,16 @@ import (
 	sdk "github.com/edgexfoundry/device-sdk-go/pkg/service"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/pelletier/go-toml"
 )
 
 const (
-	deviceNameKey     = "deviceName"
-	resourceNameKey   = "resourceName"
-	apiResourceRoute  = clients.ApiBase + "/resource/{" + deviceNameKey + "}/{" + resourceNameKey + "}"
-	handlerContextKey = "StorageHandler"
+	deviceNameKey       = "deviceName"
+	resourceNameKey     = "resourceName"
+	apiResourceRoute    = clients.ApiBase + "/resource/{" + deviceNameKey + "}/{" + resourceNameKey + "}"
+	getApiResourceRoute = clients.ApiBase + "/resource/{" + deviceNameKey + "}/{" + resourceNameKey + "}"
```

Comment:
It looks that they have same value. Is there a special use of `getApiResourceRoute`?

Reply:
Nice observation, I'll reuse the old variable.

Reply:
Done